asv_bench/benchmarks/benchmarks.py

```diff
@@ -217,12 +224,7 @@ def setup(self, shapes, how, axis):
         )
 
     def time_concat(self, shapes, how, axis):
-        if ASV_USE_IMPL == "modin":
-            execute(pd.concat([self.df1, self.df2], axis=axis, join=how))
-        elif ASV_USE_IMPL == "pandas":
-            execute(pandas.concat([self.df1, self.df2], axis=axis, join=how))
-        else:
-            raise NotImplementedError
+        execute(IMPL[ASV_USE_IMPL].concat([self.df1, self.df2], axis=axis, join=how))
```

Comment:
instead of accessing `IMPL` dictionary all the time we need to determine a module, we can just set the global variable called like `module`, and then use it inside test functions something like this:
```python
module.DataFrame...
module.concat...
```

Reply:
taking an element from a dictionary is negligible and does not affect performance

Reply:
the more this approach clearly speaks of the changeability of the implementation

Reply:
isn't the approach with `module` looks cleaner? using the `module.DataFrame` also speaks about its changeability, since we're not using concrete module name here